packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -3785,7 +3851,7 @@ function writeAssets(assets: { filePath: string; content: string }[]) {
 function mockUploadWorkerRequest(
   options: {
     available_on_subdomain?: boolean;
-    expectedEntry?: string;
+    expectedEntry?: string | RegExp;
```

Comment:
```suggestion
    expectedEntry?: string;
```
Let's not add this until we use it